README.md

```diff
@@ -67,7 +67,9 @@ To easily build on windows we can use [MSYS2](http://sourceforge.net/projects/ms
         $ pacman -S mingw-w64-i686-toolchain
```

Comment:
Don't forget to mention `mingw-w64-x86_64-toolchain` for people trying to use 64-bit Rust.